mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_folder_mute_config_test .go

```diff
@@ -0,0 +1,73 @@
+package securitycenterv2_test
```

Comment:
There's a trailing white space in this file name.

Reply:
addressed